src/content/nft/index.md

```diff
@@ -0,0 +1,175 @@
+
+<InfoBanner emoji=":eyes:">
+  <div>Check out Ethereum games, powered by NFTs...</div>
+  <ButtonLink to="/dapps?category=gaming">Explore NFT games</ButtonLink>
+</InfoBanner>
+
+### Physical items
+
+...
+
+### Making ethereum addresses more memorable
+
+One of the most useful NFTs developed so far is the Ethereum Name Service. This uses an NFT to provide your Ethereum address with an easier-to-remember name like `mywallet.eth`. This means you could ask someone to send you ETH via `mywallet.eth` rather than `0x123456789.....`.
+
+This works in a similar way to a website domain name which makes an IP address more memorable. And like domains, ENS names have value, usually based on length and relevance. With ENS you don't need a domain registry in order to facilitate the tranfer of ownership. Instead, ENS names can be traded on NFT marketplaces with ease.
+
+ENS names can also point to websites – not just wallets. For example, if you visit ethereum.eth in a web3 browser, it will open up a version of this website.
+
+## NFTs and Defi
+
+NFT-backed loans.
+Minting insurance cover as an NFT that's tradeable on secondary markets....
+
+[More on defi](/defi/)
+
+## Ethereum and NFTs
+
+Ethereum is the perfect foundation for NFTs for a number of reasons:
+
+- Transaction history and token metadata is publicly verifiable – it's simple to prove ownership history.
+- Once a transaction is confirmed, it's nearly impossible to manipulate that data to "steal" ownership.
+- Trading NFTs can happen peer-to-peer without needing platforms that can take large cuts as compensation.
+- All Ethereum products share the same "backend". Put another way, all Ethereum products can easily understand each other – this makes NFTs portable across products. You can buy an NFT on one product and sell it on another easily. As a creator you can list your NFTs on multiple products at the same time – every product will have the most up-to-date ownership information.
+- Ethereum never goes down, meaning your tokens will always be available to sell.
+
+<!-- ## How to NFT
+
+### Buy NFTs
+
+- Foundation
+- OpenSea
+- Rarible
+
+Want to show off your collection, we recommend Rainbow wallet. They even support rich media NFTs.
+
+<>
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">🎶 hear your NFTs 🌈🔜✨🤫𝙥𝙡𝙪𝙨 3⃞🅳 <a href="https://t.co/IasBswM2fT">pic.twitter.com/IasBswM2fT</a></p>&mdash; Rainbow (@rainbowdotme) <a href="https://twitter.com/rainbowdotme/status/1361161839099445251?ref_src=twsrc%5Etfw">February 15, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+</>
+
+### Mint NFTs
+
+Want to turn your content into a NFT?
+
+- Foundation
+- Other platforms... -->
+
+### Build with NFTs
+
+NFTs are built using a consistent standard known as [ERC-721](/developers/docs/standards/tokens/erc-721/). Read more about it in our docs.
+
+## NFT apps
+
+[Add links to NFT apps categorised in the same way as the dapps page]
```

Comment:
@ryancreatescopy Did you want to link directly to projects? Or linking to each "category" on the `/dapps` page?

Reply:
directly to projects. One less step if users want to go straight there rather than explore the dapps from the dapps page